contracts/UFragments.sol

```diff
@@ -144,6 +144,15 @@ contract UFragments is ERC20Detailed, Ownable {
         emit Transfer(address(0x0), owner_, _totalSupply);
     }
 
+    /*
+     * @notice A multi-chain AMPL interface method. On the base-chain contract it reflects
+     *         the value in the contract store.
+     * @return globalAMPLSupply The current total supply.
+     */
+    function globalAMPLSupply() external view returns (uint256) {
```

Comment:
I'd prefer if the base AMPL version can exist as it is, without its interface being influenced by the need of satellite chains' amples. It also duplicates the totalSupply() function. Is this required by the bridge code?

Same with the policy.

Reply:
Makes sense 👍  Removing this